docs/resources/iotda_dataforwarding_rule.md

```diff
@@ -137,19 +133,21 @@ The `dis_forwarding` block supports:
 
 * `region` - (Required, String) Specifies the region to which the DIS stream belongs.
 
-* `project_id` - (Required, String) Specifies the project ID to which the DIS stream belongs.
-
 * `stream_id` - (Required, String) Specifies the DIS stream ID.
 
+* `project_id` - (Optional, String) Specifies the project ID to which the DIS stream belongs.
+If omitted, the default project in the region will be used.
```

Comment:
pls add two spaces at the beginning of the sentence.

Reply:
done. but suggest just adding one space in the future,  as the default style of vscode